### PR TITLE
Add Identity getUrlVariables API reference for React Native platform

### PR DESCRIPTION
**Pull request by kevinlind**
Add API reference for Identity.getUrlVariables on React Native platform.
Add "Visitor Tracking Between an App and Mobile Web" section in Identity.

### CODE REVIEW

File: using-mobile-extensions/mobile-core/identity/README.md [hunk 1]
```diff
@@ -131,8 +132,128 @@ initSDK() {
 {% endtabs %}
 
 {% hint style="info" %}
-Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each client company in the Adobe Experience Cloud and is similar to the following value:`016D5C175213CCA80A490D05@AdobeOrg`. The trailing `@AdobeOrg` is required.
+Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is similar to the following value:`65002602839283012066007008864929833047`.
```
Comment (kevinlind) [inline using-mobile-extensions/mobile-core/identity/README.md]: This was incorrect. The ECID is the visitor ID not the Org ID.

File: using-mobile-extensions/mobile-core/identity/README.md [hunk 2]
```diff
@@ -131,8 +132,128 @@ initSDK() {
...
+
+{% hint style="info" %}
+
+App visitor IDs persist through upgrades.
+
+{% endhint %}
+
+#### Visitor IDs in the Mobile Web
+
+Typical mobile web implementations use the same standard Analytics `s_code.js` or `AppMeasurement.js` that is used within desktop sites. The JavaScript libraries have their own methods of generating unique visitor IDs, which causes a different vistior ID to be generated when you open mobile web content from your app.
```
Comment (kevinlind) [inline using-mobile-extensions/mobile-core/identity/README.md]: Are the names of the JavaScript libraries, `s_code.js` and `AppMeasurement.js`, still accurate and current?

File: using-mobile-extensions/mobile-core/identity/README.md [hunk 3]
```diff
@@ -131,8 +132,128 @@ initSDK() {
...
+        //For example, open the URL on the device browser        
+        //        
+        Intent i = new Intent(Intent.ACTION_VIEW);        
+        i.setData(Uri.parse(urlWithAdobeVisitorInfo));        
+        startActivity(i);    
+    }
+});
+```
+
+Alternately, starting with SDK version 1.4.0, you can call [getUrlVariables](./identity-api-reference#geturlvariables-java) and build your own URL:
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/identity/README.md]: should we mention also mention (Identity version 1.1.0) ?

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md [hunk 4]
```diff
@@ -329,7 +337,7 @@ Identity.getUrlVariables(new AdobeCallback<String>() {
 {% endtab %}
 
 {% tab title="iOS" %}
-### getUrlVariables
+### getUrlVariables<a id="geturlvariables-ios"></a>
 
 _added in ACPIdentity v2.1.0_
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/identity/identity-api-reference.md]: Let's also update this ACPCore v2.3.0

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md [hunk 5]
```diff
@@ -276,23 +276,31 @@ ACPIdentity.append(to:URL(string: "www.myUrl.com"), withCallback: {(appendedURL)
...
 {% endtabs %}
 
 ## Get visitor data as URL query parameter  <a id="getUrlVariablesTitle"></a>
 
 {% tabs %}
 {% tab title="Android" %}
-### getUrlVariables
+### getUrlVariables<a id="geturlvariables-java"></a>
 
 _added in Identity v1.1.0_
```
Comment (emdobrin) [inline using-mobile-extensions/mobile-core/identity/identity-api-reference.md]: Let's update this to sdk-core 1.4.0 or add both versions (identity and sdk-core)

File: using-mobile-extensions/mobile-core/identity/README.md (hunk 2 above)
Comment (himanshuphalak) [reply using-mobile-extensions/mobile-core/identity/README.md]: Yes, s_code.js I believe was older name of Analytics web library but AppMeasurement.js is still current.

Comment (emdobrin) [reply using-mobile-extensions/mobile-core/identity/README.md]: Thanks @himanshuphalak

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md (hunk 4 above)
Comment (kevinlind) [reply using-mobile-extensions/mobile-core/identity/identity-api-reference.md]: I'm trying to figure out wording here so it's not confusing. 
Something like `added in ACPIdentity v2.1.0 (bundled in ACPCore v2.3.0)` ?
Or more simply `added in ACPIdentity v2.1.0/ACPCore v2.3.0`?

File: using-mobile-extensions/mobile-core/identity/README.md (hunk 2 above)
Comment (kevinlind) [reply using-mobile-extensions/mobile-core/identity/README.md]: Thanks @himanshuphalak !

**Comment by kevinlind** [comment]
@emdobrin Updated the _added in_ statements to include the core bundle versions and changed ECID description to state it is a 32 character length ID.

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md (hunk 4 above)
Comment (emdobrin) [reply using-mobile-extensions/mobile-core/identity/identity-api-reference.md]: I think `added in ACPCore v2.3.0 (ACPIdentity v2.1.0)` works 👍

File: using-mobile-extensions/mobile-core/identity/README.md [hunk 6]
```diff
@@ -132,7 +132,7 @@ initSDK() {
 {% endtabs %}
 
 {% hint style="info" %}
-Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is similar to the following value:`65002602839283012066007008864929833047`.
+Previously known as MCID, the Experience Cloud ID or ECID uniquely identifies each visitor in the Adobe Experience Cloud and is a 32 character length ID.
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/identity/README.md]: - Please replace "or ECID" with (ECID)
- Please change "32 character" to "32-character"
- Can we remove "length" and just say "32-character ID"?

File: using-mobile-extensions/mobile-core/identity/README.md [hunk 7]
```diff
@@ -217,7 +217,7 @@ NSURL* url = [[NSURL alloc] initWithString:@"www.myUrl.com"];
 }];
 ```
 
-Alternately, starting with SDK version 2.3.0, you can call [getUrlVariables](./identity-api-reference#geturlvariables-ios) and build your own URL:
+Alternately, starting with SDK version 2.3.0 (ACPIdentity version 2.1.0), you can call [getUrlVariables](./identity-api-reference#geturlvariables-ios) and build your own URL:
```
Comment (rekrishn) [inline using-mobile-extensions/mobile-core/identity/README.md]: Why is this text italicized?

Comment (kevinlind) [reply using-mobile-extensions/mobile-core/identity/README.md]: Interesting. Typora uses "objective-c" for syntax highlighting in code blocks, but it messes with the parsing in GitHub's diff.  Looks like other places in our docs use "objectivec" instead, so I changed it to that.

**Comment by kevinlind** [comment]
@rekrishn I've updated the docs based on your review. Can you take a look? Thanks.

**Pull request merged**